engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb

```diff
@@ -155,10 +155,14 @@ def deploy
 
       step(1) do
         @vpc = AWS::Vpc.create()
+        @type = @vpc.type
         @vpc.modify_vpc_attrs
         @vpc.wait_until(:available)
       end
 
+      if Rails.application.config.lasso_error.present?
```

Comment:
Consider doing this in the `step()` function instead of repeating for most/all steps.